crates/warg-crypto/src/hash/mod.rs

```diff
@@ -0,0 +1,68 @@
+use anyhow::Error;
+use serde::{Deserialize, Serialize};
+use std::{fmt, str::FromStr};
+
+mod dynamic;
+mod r#static;
+
+pub use digest::{Digest, Output};
+pub use sha2::Sha256;
+
+pub use dynamic::{DynHash, DynHashParseError};
+pub use r#static::Hash;
```

Comment:
:neutral_face:

Reply:
`r#static`. I think raw idents should mostly be reserved for things like codegen.

Reply:
I think it's not so bad since it's not `pub`, but I will change it if we can come up with a naming scheme that doesn't require it.